internal/transport/handler_server.go

```diff
@@ -346,7 +349,7 @@ func (ht *serverHandlerTransport) HandleStreams(startStream func(*Stream), trace
 		case <-ht.req.Context().Done():
 		}
 		cancel()
-		ht.Close()
+		ht.Close(fmt.Errorf("request is done processing"))
```

Comment:
Same here. Why is the fact that you are done processing with the request an error message. What value does it provide and to whom?